apps/prairielearn/src/pages/instructorStudents/instructorStudents.html.tsx

```diff
@@ -469,9 +466,32 @@ function StudentsCard({
           },
           emptyState: (
             <TanstackTableEmptyState iconName="bi-person-exclamation">
-              No students found. To enroll students in your course, you can provide them with a link
-              to enroll (recommended) or invite them. You can manage the self-enrollment settings on
-              the{' '}
+              No students found. To enroll students in your course, you can provide them with a{' '}
+              <OverlayTrigger
+                placement="top"
+                tooltip={{
+                  body: 'Copied!',
+                  props: { id: 'empty-state-copy-link-tooltip' },
+                }}
+                show={copiedEnrollLink}
+              >
+                <button
+                  type="button"
+                  class="btn btn-link p-0 border-0 align-baseline"
+                  onClick={handleCopyEnrollLink}
+                >
+                  link to enroll
```

Comment:
It seems strange that we show this as an option when self-enrollment is disabled.

I don't love the "link that is not actually a link but is a button that copies text" UI. v0 has what look to me to be some nice options (https://v0.app/chat/empty-state-design-fzNdwO34rjr?ref=TBSBS0):

<img width="838" height="720" alt="Image" src="https://github.com/user-attachments/assets/e4ce17fe-216b-4a04-a21a-acc8a61f1cd6" />

If we like this, obviously we can improve the copy + alignment, but I like where this is going.

Reply:
Three notes:

- Consider `text-wrap: balance`?
- You're missing a space in `yet.Share`
- This doesn't seem to consider if modern publishing is enabled. I opened the FAll 2025 course instance in Mariana's course, which does not use modern publishing. There is no invitation buttons elsewhere on the page, but the "Invite students" button still opens a modal that allows me to invite a student. This puts me into a weird place: I can't do anything with the invite because it's not using modern publishing. Backing up: even if the button is shown accidentally, it seems like we're missing backend validation to prevent sending invites when modern publishing is disabled?

Reply:
Third point: I'll put in a PR